src/components/CalendarView/CalendarMonthly.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="h-[92%]">
+    <!-- Day List -->
+    <div class="grid w-full grid-cols-7 pb-2">
+      <span
+        v-for="day in daysList"
+        class="text-center text-sm font-normal text-gray-600"
+        >{{ day }}</span
+      >
+    </div>
+
+    <!-- Date Grid -->
+    <div
+      class="grid h-full w-full grid-cols-7 grid-rows-6 border-l-[1px] border-t-[1px]"
+    >
+      <div
+        v-for="date in currentMonthDates"
+        class="h-28 overflow-scroll border-b-[1px] border-r-[1px] border-gray-200"
```

Comment:
You might have to remove `overflow-scroll`... causes following issue
<img width="789" alt="Screenshot 2024-05-29 at 4 10 40 PM" src="https://github.com/frappe/frappe-ui/assets/13928957/fecf4eec-d68b-4fa3-89e3-a619354da65d">

Reply:
What is the issue here, did not understand?

Reply:
> What is the issue here, did not understand?

Scrollbars in each cell... shouldn't it look like this.
<img width="1051" alt="Screenshot 2024-05-30 at 9 49 37 AM" src="https://github.com/frappe/frappe-ui/assets/13928957/55cee6f1-88ab-4d49-a0bc-7defa4c48399">